dev/dags/basic_cosmos_task_group.py

```diff
@@ -25,7 +25,7 @@
 
 
 @dag(
-    schedule_interval="@daily",
+    schedule="@daily",
```

Comment:
While we support Airflow 2.3, we can't move from `schedule_interval` to `schedule` 😞 
This comment applies to the remaining 9 DAGs changed.

Reply:
Aaah, that explains it, I was surprised to find `scheduel_interval` in there 😅 will change it back!